requirements/main.txt

```diff
@@ -1215,8 +1216,16 @@ pycparser==2.21 \
     --hash=sha256:8ee45429555515e1f6b185e78100aea234072576aa43ab53aefcae078162fca9 \
     --hash=sha256:e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206
     # via cffi
-pycurl==7.44.1 \
-    --hash=sha256:5bcef4d988b74b99653602101e17d8401338d596b9234d263c728a0c3df003e8
+pycurl==7.43.0.5 \
+    --hash=sha256:1957c867e2a341f5526c107c7bbc5014d6e75fdc2a14294fcb8a47663fbd2e15 \
+    --hash=sha256:50aee0469511a9708a1f1a50d510b5ec2013fc6f5e720c32bbcb3b9c7b0f45b1 \
+    --hash=sha256:667db26516e50ce4a853745906f3b149c24756d85061b9d966eb7ec43a8c48a4 \
+    --hash=sha256:7cc13d3421cbd31921d77e22d1f57c0e1a8d0fb461938a587689a93162ccef2f \
+    --hash=sha256:a0c62dbc66b9b947832307d6cf7bdb5e4da906ce7b3efe6f74292e8f3dc5abe3 \
+    --hash=sha256:a6966e8d9ccda31c6d077c4f8673aaa88141cc73d50e110e93e627b816d17fd1 \
+    --hash=sha256:beadfa7f052626864d70eb33cec8f2aeece12dfb483c2424cc07b057f83b7d35 \
+    --hash=sha256:c5c379c8cc777dda397f86f0d0049480250ae84a82a9d99d668f461d368fb39c \
+    --hash=sha256:ec7dd291545842295b7b56c12c90ffad2976cc7070c98d7b1517b7b6cd5994b3
```

Comment:
This downgraded `pycurl` because of a hard pin here: https://inspector.pypi.io/project/kombu/5.3.0/packages/8c/04/4adf5f7bdb2f00c20f09afca1ab0e9899c26f236dea3eb8354d61aa7ae02/kombu-5.3.0.tar.gz/kombu-5.3.0/requirements/extras/sqs.txt#line.2

That version of `pycurl` doesn't build for us, I didn't look into why.

This was removed in https://github.com/celery/kombu/pull/1747 but hasn't been released yet, so I think dependency resolution is going to fail here until that happens.